src/builder/QuarkBuilder.sol

```diff
@@ -402,6 +402,14 @@ contract QuarkBuilder {
             chainAccountsList = Accounts.findChainAccountsWithPaymentInfo(chainAccountsList, payment);
         }
 
+        // Initialize comet supply max flag
+        bool isMaxSupply = cometSupplyIntent.amount == type(uint256).max;
+        // Convert transferIntent to user aggregated balance
```

Comment:
Comment needs to be updated. Same for the other function

Reply:
oops yeah, will update those comments.